.github/workflows/ci.yml

```diff
@@ -160,19 +160,15 @@ jobs:
           files: ./coverage.xml
 
   publish:
-    # Build wheels when pushing to any branch except main
-    # publish.yml will only publish if tagged ^v.*
+    needs: [test, docs]
+    # Build wheels on PRs only when labelled. Releases will only be published if tagged ^v.*
+    # see https://github-actions-workflows.openastronomy.org/en/latest/publish.html#upload-to-pypi
```

Comment:
Are the other jobs not "if" then "needs" if they have both?

Reply:
That is true, if my script will fix it automatically then fine, otherwise it will have to stay due to me not having another hour to work on this.

Reply:
Narrator: "it did not"

Reply:
Looks like we will have to merge all of these and then open a whole other batch manually to fix this.